### ISSUE TITLE
Can't bundle with rollup

### ISSUE THREAD

**Issue by vukasin-nikodijevic** [michaelbromley/ngx-pagination#111]
**Angular 2 version**:
2.2.0

**ng2-pagination version**:
1.0.1

**Description of issue**:
Rollup can't find named export of Ng2PaginationModule. I've tried to force it by following rollup commonjs plugin instructions - but it still doesn't work:
https://github.com/rollup/rollup-plugin-commonjs#custom-named-exports

**Steps to reproduce**:
```
import { Ng2PaginationModule } from 'ng2-pagination';

@NgModule({
    bootstrap: [MyAppComponent],
    imports: [
        Ng2PaginationModule,
    ],
})
export class MylModule {
}
```

**Expected result**:
To bundle normally without error.

**Actual result**:
```
'Ng2PaginationModule' is not exported by node_modules/ng2-pagination/index.js (imported by module.ts). For help fixing this error see https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module
Error: 'Ng2PaginationModule' is not exported by node_modules/ng2-pagination/index.js (imported by module.ts). For help fixing this error see https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module
```

**Comment by michaelbromley** [comment]
Hi,

Thanks for the report. I've never used Rollup, so I cannot offer any specific advice. What about it you try targeting the `dist/ng2pagination.js` file directly?

**Comment by michaelbromley** [comment]
I have just published v3.0.0 (note: name changed to ngx-pagination with this release), which is now built with rollup and bundled as ES modules. Tested with a rollup-based app and now works.

Please re-open if there are any further issues I have overlooked.
